agent/host_test.go

```diff
@@ -0,0 +1,42 @@
+// +build linux
+
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package agent
+
+import (
+        "github.com/Jaganathancse/openstack-nfv-params/openstacknfv"
+	"testing"
+)
+
+func TestOpenstackNfvParams(t *testing.T) {
+
+        numa, err := openstacknfv.GetOpenstackNfvParams()
```

Comment:
I'm not against external library, but they should at least 

 1/ use mostly golang library/function
 2/ do not use shell command, as a host/program read/speak together with bytes/ABI/API ... not chars/string/quote   (I don't even talk about sudo call ...)
 3/ provide a simple API : not mixing kernel info, cpu isolations, nova info, ...
 4/ tests within Skydive should test (as much possible) your API call and implemented features